typings.d.ts

```diff
@@ -426,6 +432,7 @@ declare module 'contentful-ui-extensions-sdk' {
     LOCATION_ENTRY_SIDEBAR: string;
     LOCATION_DIALOG: string;
     LOCATION_ENTRY_EDITOR: string;
+    LOCATION_PAGE: string;
```

Comment:
We need to create one more SDK type:

```
  export type PageExtensionSDK = BaseExtensionSDK & {
    /** A set of IDs actual for the extension */
    ids: Pick<IdsAPI, 'environment' | 'space' | 'extension' | 'user'>;
  }
```

and update `init` function declaration